### DISCUSSION TITLE
How can I comment on a module that I don't like or that I don't use?[Feature Request]: Short description of the feature

### DISCUSSION THREAD

**Discussion by ablond74** [JaKooLit/Hyprland-Dots#222] [Q&A] [answered]
### Description

[Provide a clear and concise description of the feature you'd like to see added.]

### Use Case

[Describe how this feature would be useful to you or to other users of the project.]

### Proposed Solution

[If you have a specific solution in mind, describe it here. If not, you can skip this section.]

### Alternatives

[Are there any alternative solutions or workarounds that you've considered? If so, describe them here.]

### Additional Information

[Add any other information about the feature request here. For example, you might include links to similar features in other projects, or screenshots or diagrams to help explain your idea.]

**Comment by JaKooLit** [comment, marked as answer]
are you talking waybar module?

 - unwanted waybar module:
    - Simplest is to edit `~/.config/waybar/modules` . Put // before or delete that module.
    - Or you dont really need to worry about since the modules not defined on waybar configs are not loaded.
    - So best and recommended to is to edit both the configs and styles that you prefer on configs and styles folder in  `~/.config/waybar` . Then simply load it.
    
    
https://github.com/JaKooLit/Hyprland-Dots/wiki/TIPS#%EF%B8%8F-waybar-customizations
